python/examples/asset_agnostic_rules/main.py

```diff
@@ -56,20 +56,15 @@
             sub_expressions=[
                 SubExpression("voltage.overvoltage", {"$1": 75}),
                 SubExpression("voltage.undervoltage", {"$1": 30}),
-                SubExpression(
-                    "velocity.vehicle_stuck",
-                    {
-                        "$1": "vehicle_state",
-                        "$2": "mainmotor.velocity",
-                    },
-                ),
-                SubExpression(
-                    "velocity.vehicle_not_stopped",
-                    {
-                        "$1": "vehicle_state",
-                        "$2": "10",
-                    },
+                SubExpression("velocity.vehicle_not_stopped", {"$2": 10}),
+            ],
+            channel_references=[
+                RuleChannelReference("overvoltage", {"$2": "vehicle_state"}),
```

Comment:
Why does `SubExpression` include the namespace, but `RuleChannelReference` doesnt?

Reply:
That's a good shout, I forgot to mentioned this -- this issue going to be resolved in the next PR when I deprecate namespaces. So it'll be consistent when we release.